content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,106 @@ APF adds the following two headers to each HTTP response message.
 - `X-Kubernetes-PF-PriorityLevel-UID` holds the UID of the
   PriorityLevelConfiguration object associated with that FlowSchema.
 
+## Good practices for using API Priority and Fairness
+When a given priority level exceeds its permitted concurrency, requests can 
+experience increased latency or be dropped with an HTTP 429 (Too Many Requests) 
+error. To prevent these side effects of APF, you can modify your workload or 
+tweak your APF settings to ensure there are sufficient seats available to serve
+your requests.
+
+To detect whether requests are being rejected due to APF, check the following
+metrics:
+- apiserver_flowcontrol_rejected_requests_total: the total number of requests 
+rejected per FlowSchema and PriorityLevelConfiguration.
+- apiserver_flowcontrol_current_inqueue_requests: the current number of requests
+queued per FlowSchema and PriorityLevelConfiguration.
+- apiserver_flowcontrol_request_wait_duration_seconds: the latency added to
+requests waiting in queues.
+- apiserver_flowcontrol_priority_level_seat_utilization: the seat utilization 
+per PriorityLevelConfiguration.
+
+### Workload modifications {#good-practice-workload-modifications}
+To prevent requests from queuing and adding latency or being dropped due to APF, 
+you can optimize your requests by:
+
+- Reducing the rate at which requests are executed. A fewer number of requests 
+over a fixed period will result in a fewer number of seats being needed at a 
+given time.
+- Avoid issuing a large number of expensive requests concurrently. Requests can 
+be optimized to use fewer seats or have lower latency so that these requests 
+hold those seats for a shorter duration. List requests can occupy more than 1 
+seat depending on the number of objects fetched during the request. Restricting 
+the number of objects retrieved in a list request (by using pagination or 
+limiting the request to a particular namespace) will use less total seats over a
```

Comment:
nit: currently setting namespace doesn't affect the number of seats used by the request

xref: https://github.com/kubernetes/kubernetes/pull/120143#discussion_r1303488278

Reply:
Thanks for clarifying, I'll update this to only mention using pagination as an option to reduce the number of objects retrieved. Does using a label or field selector also not reduce the number of objects used in the seat calculation?

Reply:
> Does using a label or field selector also not reduce the number of objects used in the seat calculation?

Correct - neither of those matters (for now).